src/model/input-readers/git-repo.ts

```diff
@@ -14,6 +18,9 @@ export class GitRepoReader {
   }
 
   public static async GetBranch() {
+    if (Input.cloudRunnerCluster === 'local') {
+      return '';
+    }
```

Comment:
As a general note, if you have safeguard conditions that immediately branch off with a return you can also make it into a one-liner and leave an empty line after. I usually do this for very simple statements to further improve readability.